### PR TITLE
802.1x support Mojo networks AP + documentation

### PR DESCRIPTION
**Pull request by lzammit**
# Description

802.1x support for the cloud controller Mojo Networks. It has been tested with C-65 access point.
# Impacts
- Documentation
- Switch modules
# Delete branch after merge

YES
# NEWS file entries

802.1x support for Mojo Networks

### CODE REVIEW

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -4779,6 +4779,82 @@ So, since the matching is done using the Firewall Filter ID configuration field,
 
 NOTE: You need to have the *Per-User Firewall* license in order to benefit this feature.
 
+Mojo Networks
+^^^^^^^^^^^^^
+
+PacketFence supports only SSIDs configured with 802.1x.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: 802.1x -> 802.1X everywhere

File: lib/pf/Switch/Mojo.pm [hunk 2]
```diff
@@ -0,0 +1,245 @@
+package pf::Switch::Mojo;
+=head1 NAME
+
+pf::Switch::Cisco::WLC - Object oriented module to parse SNMP traps and manage
```
Comment (extrafu) [inline lib/pf/Switch/Mojo.pm]: WLC?

File: lib/pf/Switch/Mojo.pm [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+Cisco Wireless Controllers (WLC) and Wireless Service Modules (WiSM)
```
Comment (extrafu) [inline lib/pf/Switch/Mojo.pm]: Cisco?

File: lib/pf/Switch/Mojo.pm [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+    $add_attributes_ref = {} if (!defined($add_attributes_ref));
+
+    if (!defined($self->{'_radiusSecret'})) {
+        $logger->warn(
+            "Unable to perform RADIUS CoA-Request on (".$self->{'_id'}."): RADIUS Shared Secret not configured"
+        );
+        return;
+    }
+
+    $logger->info("deauthenticating");
```
Comment (extrafu) [inline lib/pf/Switch/Mojo.pm]: bogus log statement

File: lib/pf/Switch/Mojo.pm [hunk 5]
```diff
@@ -0,0 +1,243 @@
+package pf::Switch::Mojo;
+=head1 NAME
+
+pf::Switch::Mojo - Object oriented module to return radius attributes to a Mojo Networks Acces Point.
```
Comment (fdurand) [inline lib/pf/Switch/Mojo.pm]: Access

**Comment by extrafu** [comment]
Update this PR ASAP so it gets merged.

**Pull request merged**